src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/multiples/MultiplesSendNotificationController.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers.multiples;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.et.common.model.multiples.MultipleCallbackResponse;
+import uk.gov.hmcts.et.common.model.multiples.MultipleData;
+import uk.gov.hmcts.et.common.model.multiples.MultipleDetails;
+import uk.gov.hmcts.et.common.model.multiples.MultipleRequest;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.multiples.MultiplesSendNotificationService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.CallbackRespHelper.getMultipleCallbackRespEntity;
+
+@Slf4j
+@RequestMapping("/multiples/sendNotification")
+@RestController
+@RequiredArgsConstructor
+public class MultiplesSendNotificationController {
+    private final MultiplesSendNotificationService multiplesSendNotificationService;
+
+    /**
+     * send Notification about to start.
+     *
+     * @param multipleRequest holds the request and case data
+     * @return Callback response entity with case data attached.
+     */
+    @PostMapping(value = "/aboutToStart", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "aboutToStart")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully",
+            content = {
+                @Content(mediaType = "application/json",
+                        schema = @Schema(implementation = MultipleCallbackResponse.class))
+            }),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<MultipleCallbackResponse> aboutToStart(@RequestBody MultipleRequest multipleRequest) {
+
+        List<String> errors = new ArrayList<>();
+        // TODO: Get hearing details from lead case as part of RET-4711
+
+        log.warn("About to start");
```

Comment:
any reason for this being a warning?

Reply:
It's just something I had to test locally. It's not needed and this callback will be completed in another story. Removed for now